### PR TITLE
Gift Entry: Wire up review donations button

### PR DESCRIPTION
**Pull request by psadie-sfdo**
# Critical Changes

# Changes
- Adds the review donations component to the form renderer component
# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: src/lwc/geFormRenderer/geFormRenderer.js [hunk 1]
```diff
@@ -560,4 +588,116 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
...
+                    }
+                );
+            }
+        );
+
+        // Workaround to force rerendering of the form.
+        let that = this;
+        this.sections = [];
+        // eslint-disable-next-line @lwc/lwc/no-async-operation
+        setTimeout(() => {
```
Comment (ptewson-sfdo) [inline src/lwc/geFormRenderer/geFormRenderer.js]: :(

File: src/aura/GE_GiftEntry/GE_GiftEntryController.js [hunk 2]
```diff
@@ -43,25 +43,18 @@
...
-            component.set('v.isLoading', true);
-        });
+        helper.handleCloseModal(component);
     },
 
     /*******************************************************************************
     * @description Handles receipt of events from utilDedicatedListener component
     * and notifies the geTemplates component.
     */
     handleBatchWizardEvent: function (component, event, helper) {
```
[the remark this answers is not in the document]
Comment (psadie-sfdo) [reply src/aura/GE_GiftEntry/GE_GiftEntryController.js]: Yep, we still need this one for the batch wizard in the landing page.

**Comment by psadie-sfdo** [comment]
**lurch: attach W-038940

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HrCSQA0" target="_blank">W-038940</a>

**Pull request merged**